.vscode/launch.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
this file should be removed :)

Reply:
ok - apologies - I didn't intend to commit it.